app.go

```diff
@@ -0,0 +1,183 @@
+package appcenter
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+)
+
+// App ...
+type App struct {
+	client      Client
+	owner, name string
+}
+
+// NewRelease ...
+func (a App) NewRelease(filePath string, opts ...ReleaseOptions) (Release, error) {
+	// send file upload request
+	var (
+		postURL      = fmt.Sprintf("%s/v0.1/apps/%s/%s/release_uploads", baseURL, a.owner, a.name)
```

Comment:
Is URL encoding needed for params? url.URL can also be used to construct an URL.

Reply:
it is done by newrequest